src/components/Creepypasta.spec.js

```diff
@@ -2,10 +2,11 @@ import { render, screen } from '@testing-library/react';
 import Creepypasta from './Creepypasta.js';
 
 describe('Creepypasta', () => {
```

Comment:
I think this test fails because you are calling the component without MemoryRouter.